templates/ros2_control/controller/dummy_chainable_controller.cpp

```diff
@@ -139,6 +143,30 @@ controller_interface::CallbackReturn DummyClassName::on_configure(
   return controller_interface::CallbackReturn::SUCCESS;
 }
 
+void DummyClassName::reference_callback(const std::shared_ptr<ControllerReferenceMsg> msg)
```

Comment:
saw that ref_callback is called in different order in standard and chainable controller, hence adjusting it to be consistent with standard controller

Reply:
in the chainable controller,
 this callback is not used at all. I mean, you can call it, but nothing will happen.

Reply:
sorry for the confusion i should have written "ref_callback is defined in different order in standard and chain-able controller classes, hence adjusting it to be consistent with standard controller"

its trivial change, just changed the order